src/drv_oculus_rift/rift.c

```diff
@@ -239,6 +239,11 @@ static ohmd_device* open_device(ohmd_driver* driver, ohmd_device_desc* desc)
 	// apply sensor config
 	set_coordinate_frame(priv, priv->coordinate_frame);
 
+	// Turn the screens on
+	// TODO: this should probably be done conditionnally for the CV1
```

Comment:
Oh okay, it would be good to have so we don't send the wrong message to devices. Maybe as a enum field on the rift_priv struct?